assets/src/dashboard/components/card-grid/index.js

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const GridContainer = styled.div`
+  display: grid;
+  width: ${({ theme }) => theme.grid.desktop.width};
+  grid-template-columns: repeat(
+    ${({ theme }) => theme.grid.desktop.columns},
+    1fr
```

Comment:
Based on the designs I don't think we'd ever want variable fractional relations, but it might be good to expose the option.